### ISSUE TITLE
False positive for rule 942190

### ISSUE THREAD

**Issue by sdawes437** [coreruleset/coreruleset#4312]
### Description

This application contains a feature that allow an authenticated actor to send messages to another authenticated actor, of a different type. Note this bug is not unique to this feature, and seems to be related to the 'having friends over is nice string' in the request body.

### How to reproduce the misbehavior (-> curl call)

```
curl 'https://foo.com/encryption/encrypt' \
  -H 'accept: application/json' \
  -H 'accept-language: en-GB,en-US;q=0.9,en;q=0.8' \
  -H 'authorization: Bearer xxxx' \
  -H 'content-type: application/json' \
  -H 'origin: https://foo.com' \
  -H 'referer: https://foo.com/' \
  -H 'user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/141.0.0.0 Safari/537.36' \
  --data-raw '[{"bodies":["having friends over is nice"],"source":{"@id":"some-uuid","@type":"MailboxConversation"}}]'
```

Note the requests are not blocked with other state

### Your Environment

* CRS version v3.3.2
* Paranoia level setting PL1)
* ModSecurity version (e.g., 2.9.6): N/A (Using Google Cloud Armour with these rules)
* Web Server and version or cloud provider / CDN (e.g., Apache httpd 2.4.54): Google Cloud Platform
* Operating System and version: N/A (web application firewall using managed service, Google Cloud Armour)

### Confirmation

[x] I have removed any personal data (email addresses, IP addresses,
    passwords, domain names) from any logs posted.

**Comment by airween** [comment]
Hi @sdawes437,

thanks for reporting this issue.

> This application contains a feature that allow an authenticated actor to send messages to another authenticated actor, of a different type. Note this bug is not unique to this feature, and seems to be related to the 'having friends over is nice string' in the request body.

Sorry to hear you ran into this problem.

> * CRS version v3.3.2

Unfortunately 3.3.2 is a bit old - this was released in June, 2021, and it does not have any support.

I'm sure that's very annoying, but what you can do is:

* create an exclusion, see our [docs](https://coreruleset.org/docs/2-how-crs-works/2-3-false-positives-and-tuning/)
* upgrade you CRS to 3.3.7 - it seems like there is no any FP:

```
curl -v -H "x-crs-version: 3.3.7" \
    -H "x-format-output: txt-matched-rules" \
    -H "x-crs-paranoia-level: 1" \
    -H 'accept: application/json' \
    -H 'accept-language: en-GB,en-US;q=0.9,en;q=0.8' \
    -H 'authorization: Bearer xxxx' -H 'content-type: application/json' \
    -H 'origin: https://sandbox.coreruleset.org' \
    -H 'referer: https://sandbox.coreruleset.org' \
    -H 'user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/141.0.0.0 Safari/537.36' \
    --data-raw '[{"bodies":["having friends over is nice"],"source":{"@id":"some-uuid","@type":"MailboxConversation"}}]' \
    'https://sandbox.coreruleset.org'
```

Note: you can increase the `x-crs-paranoia-level` value, but I was able to get any triggered rule on PL4.

Note: Sandbox does not support 3.3.2, only 3.3.7.

**Comment by sdawes437** [comment]
Thanks @airween - it seems that unfortunately Cloud Armour only supports up to 3.3.2 for some reason. It looks like excluding the rules maybe the only option available, unfortunately
